lib/api.rb

```diff
@@ -0,0 +1,20 @@
+module Api
```

Comment:
@jrafanie I guess I like the directory structure - controllers in https://github.com/ManageIQ/manageiq/pull/9868 will be in the least-surprising app/controllers/api directory. Now that you've said this though I have a growing uneasiness with it. `MiqApi`? `VmdbApi`?